portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase

```diff
@@ -540,4 +540,23 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-196410. A User can access the WebDAV link from Documents and Media, but a prompt has to be displayed about the password that can be created at My Account."
+	@priority = 4
+	test BluePromptInDocumentsAndMediasAccessFromDesktop {
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given: A User navigates to Documents and Media in Site Menu") {
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "test-site-name");
+		}
+
+		task ("When: Clicks on the Kebab Menu and then on Access from Desktop, the WebDAV link and the notification prompt should be displayed") {
+			Portlet.gotoPortletOptions(portletOption = "Access from Desktop");
+		}
+
+		task ("Then: Log out") {
+			User.logoutPG(); 
```

Comment:
Can we add additional assertion steps here?

Reply:
Assertion steps added